testsuite/integration-tests/src/test/resources/arquillian.xml

```diff
@@ -27,6 +27,18 @@
                 <property name="managementPort">${container.management.port.manual.gzip}</property>
             </configuration>
         </container>
+        <container qualifier="${container.qualifier.managed.encoding}" mode="class">
```

Comment:
`mode="class"` is wrong here. We can't start this container repeatedly for each test class in TS (~491 times in integration-tests)

Reply:
What should be the proper mode? Manual?

Reply:
Manual LGTM

Reply:
+1